setup.py

```diff
@@ -261,7 +261,7 @@ def run(self):
         log.info("Building Fiona for gdal 1.x: {0}".format(gdalversion))
         ext_modules.append(
             Extension('fiona._shim', ['fiona/_shim1.c'], **ext_options))
-    else:
+    elif gdal_minor_version == 2:
```

Comment:
This needs to be `gdal_major_version` instead of `gdal_minor_version` I think

Reply:
oh no! yes, that was my mistake!